llvm/docs/LangRef.rst

```diff
@@ -23475,6 +23475,86 @@ Examples:
       %active.lane.mask = call <4 x i1> @llvm.get.active.lane.mask.v4i1.i64(i64 %elem0, i64 429)
       %wide.masked.load = call <4 x i32> @llvm.masked.load.v4i32.p0v4i32(<4 x i32>* %3, i32 4, <4 x i1> %active.lane.mask, <4 x i32> poison)
 
+.. _int_experimental_get_alias_lane_mask:
+
+'``llvm.get.alias.lane.mask.*``' Intrinsics
```

Comment:
```suggestion
'``llvm.experimental.get.alias.lane.mask.*``' Intrinsics
```
?

Reply:
Other sections have the s on the end of Intrinsic, but thanks for spotting the wrong intrinsic name. Fixed that now.